app/static/js/tilt_graph_socketio.js

```diff
@@ -0,0 +1,78 @@
+Highcharts.setOptions({
+  time: {
+      useUTC: false
+  }
+});
+
+Highcharts.chart(graph_data.chart_id, {
+chart: {
+  events: {
+    load: function () {
+      var self = this
+      var event_name = 'tilt_session_update|' + graph_data.chart_id
+      socket.on(event_name, function (event)
+      {
+          var data = JSON.parse(event);
+          if (data['voltage']) {
+            self.setTitle(graph_data.title, {text: 'Voltage: ' + data['voltage'] + 'V'});
```

Comment:
Don't think we have voltage in the broadcast from tilt do we?

If not think this is better suited for device Color

Reply:
no voltage. see my latest merge